iyzipay/iyzipay_resource.py

```diff
@@ -68,7 +68,10 @@ def format_header_string(api_key, hashed):
 
     @staticmethod
     def resource_pki(request):
-        return 'locale=' + request.get('locale') + ',conversationId=' + request.get('conversationId') + ','
+        if request.get('conversationId') != None:
+            return 'locale=' + request.get('locale') + ',conversationId=' + request.get('conversationId') + ','
+        else:
+            return 'locale=' + request.get('locale') + ','
```

Comment:
how about ?
```python
return 'locale=' + request.get('locale') + (',conversationId=' + request.get('conversationId') + ',' if request.get('conversationId') else ',')
```

Reply:
Yes that's better, thanks. I'm going to update my PR.